cmd/fleet/create_fleet.go

```diff
@@ -30,11 +28,12 @@ func NewCmdCreateFleet(config *cfg.Config) *cobra.Command {
 			ctx := cmd.Context()
 
 			var err error
-			in.Config, err = readConfig(configFile, configFormat)
+			in.RawConfig, err = readConfig(configFile)
 			if err != nil {
 				return err
 			}
 
+			in.ConfigFormat = types.ConfigFormat(configFormat)
```

Comment:
I think there shold be a smart way to parse the config format like before.
If the configFormat is empty or auto, it should be taken from the file extension.

```go
if format == "" || strings.ToLower(format) == "auto" {
	format = strings.TrimPrefix(filepath.Ext(filename), ".")
}
```